pingcap/tidb
# topsql: fix the issue of TiDB doesn't set the request tag for DDL before send RPC request to TiKV

### What problem does this PR solve?


Issue Number: close #33132

Problem Summary: This PR fix the issue of TiDB doesn't set the request tag for DDL before send rpc to TiKV.

### What is changed and how it works?

Set the DDL request tag.

Other problem: After this PR, TopSQL still doesn't contain the `scan_row`/`scan_index` data , I will fix this in next PR.

### Check List

Tests 


1. create a table and insert 10000000 rows data.

```sql
CREATE TABLE `t_full_table_scan` (
  `a` bigint(20) NOT NULL,
  `b` bigint(20) DEFAULT NULL,
  `c` timestamp(6) NULL DEFAULT NULL,
  `d` varchar(100) DEFAULT NULL,
  `e` decimal(30,8) DEFAULT NULL,
  PRIMARY KEY (`a`)
);
```

3. execute following DDL:

```sql
alter table t_full_table_scan add index idx2 (c,d,b);
alter table t_full_table_scan modify column e decimal(30, 8);
```
4. check the topsql UI in tidb-dashboard.

TiDB:
![image](https://user-images.githubusercontent.com/26020263/158534268-f5103da1-6efa-4e93-adae-d5713aa45668.png)

TiKV:
![image](https://user-images.githubusercontent.com/26020263/158534347-d422a7ed-a026-402f-a241-5388fc5b8308.png)

### Release note



```release-note
None
```
